src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,296 @@
+import {
+  canInteract,
+  Item,
+  Location,
+  Monster,
+  runChoice,
+  visitUrl,
+} from "kolmafia";
+import { get } from "../../property";
+import { $item, $location, $monster } from "../../template-string";
+
+const radio = "place.php?whichplace=airport_spooky&action=airport2_radio";
+const omegapanel =
+  "place.php?whichplace=airport_spooky_bunker&action=si_controlpanel";
+
+export function have(): boolean {
+  return get("spookyAirportAlways");
+}
+
+class QuestData {
+  name: string;
+  reward: number;
+  questStateProperty: string;
+  questProgressLimit: number;
+  questProgressProperty: string;
+  canUseWanderers: boolean;
+  canUseRunaways: boolean;
+  cleeshsafe: boolean;
+  questLocation: Location;
+  requiredItem: Item;
+  maximise: string;
+
+  /**
+   * Process for determining where to put a wanderer to extract additional value from it
+   * @param name easy to refer to name of the quest
+   * @param reward how many coinspiracy we get for completing the quest
+   * @param questStateProperty name of the mafia property tracing the current state of the quest (e.g. unstarted, step[x], finished)
+   * @param questProgressLimit Maximum questProgressProperty can reach
+   * @param questProgressProperty name of the mafia property that tracks progress of the current quest state
+   * @param canUseWanderers whether wandering monsters can be used to progress the quest
+   * @param canUseRunaways whether runaways can be used to progress the quest
+   * @param cleeshsafe whether cleeshes will advance the quest
+   * @param questLocation where to adventure to progress the quest
+   * @param requiredItem the required item (if any) needed to be equipped to progress the quest
+   */
+  constructor(
+    name: string,
+    reward: number,
+    questStateProperty: string,
+    questProgressLimit: number,
+    questProgressProperty: string,
+    canUseWanderers: boolean,
+    canUseRunaways: boolean,
+    cleeshSafe: boolean,
+    questLocation: Location,
+    requiredItem: Item,
+    maximise: string
+  ) {
+    this.name = name;
+    this.reward = reward;
+    this.questStateProperty = questStateProperty;
+    this.questProgressLimit = questProgressLimit;
+    this.questProgressProperty = questProgressProperty;
+    this.canUseWanderers = canUseWanderers;
+    this.canUseRunaways = canUseRunaways;
+    this.cleeshsafe = cleeshSafe;
+    this.questLocation = questLocation;
+    this.requiredItem = requiredItem;
+    this.maximise = maximise;
+  }
+
+  currentQuest(): boolean {
```

Comment:
I would rename this to `isCurrentQuest` if we're continuing to make this a class